packages/api/apollo-server/queries/users/index.ts

```diff
@@ -0,0 +1,10 @@
+export default async () => {
+  return [
+    {
+      firstName: 'Alice',
+      lastName: 'A',
+      utorID: 'alice01',
+      userType: 'STUDENT'
+    }
```

Comment:
Does this return all the users? some of the users? If I want to make a function to get user by id, where would that go?